src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -36,6 +36,10 @@
  */
 class TranslationUpdateCommand extends Command
 {
+    protected const ASC = 'asc';
+    protected const DESC = 'desc';
+    protected const SORT_ORDERS = [self::ASC, self::DESC];
```

Comment:
`const`s should be `private` (the class is marked as `final` anyway).

Reply:
done